README.md

```diff
@@ -1,21 +1,38 @@
+# ScyllaDB JS Driver
+
 ![Static Badge](https://img.shields.io/badge/API%20coverage-0%25-red)
 ![test workflow](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/actions/workflows/tests.yml/badge.svg?branch=main)
 ![quality workflow](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/actions/workflows/code-quality.yml/badge.svg?branch=main)
 
-# ScyllaDB JS Driver
+This is a client-side driver for [ScyllaDB](https://www.scylladb.com/) written in NodeJs.
+This driver is an overlay over the [ScyllaDB Rust Driver](https://github.com/scylladb/scylla-rust-driver),
+with the interface based on the [DataStax Node.js Driver](https://github.com/datastax/nodejs-driver).
+Although optimized for ScyllaDB, the driver is also compatible with [Apache Cassandra®].
 
-Documentation available [here](https://scylladb-zpp-2024-javascript-driver.github.io/scylladb-javascript-driver/docs)
+This diver is currently in the experimental state.
 
-Book available [here](https://scylladb-zpp-2024-javascript-driver.github.io/scylladb-javascript-driver/book)
+Documentation is available [here](https://scylladb-zpp-2024-javascript-driver.github.io/scylladb-javascript-driver/docs) <!-- Note: update those links after first commit to new main -->
+
+Book is available [here](https://scylladb-zpp-2024-javascript-driver.github.io/scylladb-javascript-driver/book)
```

Comment:
:wrench: Please briefly describe what is/should be in the documentation and what is/should be in the Book.